libraries/consul_service.rb

```diff
@@ -55,6 +55,7 @@ def default_environment
         {
           'GOMAXPROCS' => [node['cpu']['total'], 2].max.to_s,
           'PATH' => '/usr/local/bin:/usr/bin:/bin',
+          'SHELL' => node['consul']['service_shell'],
```

Comment:
@Ginja Yeah, that looks good. Could you please update your PR by adding a condition?